documentation/get-started/tutorials/get-started-qfs.en.md

```diff
@@ -93,6 +93,11 @@ Layer
     layer like a WMS or to take a working copy of an unsupported format
     like an ECW or MrSID layer.
 
+!!! note
+    It is only possible with a PNG raster with a PNW world file at the moment.
```

Comment:
Is this iOS specific (as the pull request title suggests)? If yes, I think this needs to be mentioned.